### PR TITLE
Add Siri Extension Localization for 7.0

### PR DESCRIPTION
**Pull request by oliviabrown9**

### CODE REVIEW

File: templates/focus-ios.xliff [hunk 1]
```diff
@@ -28,6 +28,19 @@
       </trans-unit>
     </body>
   </file>
+  <file original="Blockzilla/en.lproj/Intents.strings" source-language="en" datatype="plaintext">
+    <header>
+      <tool tool-id="com.apple.dt.xcode" tool-name="Xcode" tool-version="9.1" build-num="9B55"/>
+    </header>
+    <body>
+      <trans-unit id="Erase">
```
Comment (Delphine) [inline templates/focus-ios.xliff]: Could we get a short comment here, just specifying if it's a noun or a verb? (I'm assuming it's a verb, but better to be extra clear in those cases ;) ).
Also, not sure this is a valid ID?

File: templates/focus-ios.xliff [hunk 2]
```diff
@@ -28,6 +28,19 @@
...
+        <source>Erase</source>
+      </trans-unit>
+      <trans-unit id="Erase browsing session">
```
Comment (Delphine) [inline templates/focus-ios.xliff]: this doesn't look like a valid ID (should probably be more something that looks like "Erase.Browsing.Session")

**Comment by oliviabrown9** [comment]
@st3fan Do you have any ideas why the export script isn't picking up comments? I also noticed that this file has a different file path (Blockzilla/en.lproj/... vs Blockzilla/...) than other strings files. I'm not sure if this is contributing to any of the issues. Lastly, the intent definition file only supports two types of keys: the word itself or an identifier generated by Xcode (random alphanumeric id). Happy to use whichever you recommend.

**Comment by oliviabrown9** [comment]
@st3fan The comments are the same as we have for other .strings files, such that it should pick up a comment above the string as the note. For example, this works with Intro.strings.

**Comment by Delphine** [comment]
Leaving for the day now, and I unfortunately won't have time to look into this tomorrow before I leave on PTO. Flagging @flodolo and @gueroJeff so they can pick this up if needed, since they're covering up for me during my absence.

Note that I was told sign-offs on localization for Focus iOS v7 was supposed to be this upcoming Monday. I think it's unrealistic to have something localized and tested over a weekend, unfortunately.

I'll let Product team decide what to do going forwards. thanks!

**Comment by flodolo** [comment]
@Sdaswani
Are you able to give an update on this? I don't know if Olivia is still around, and who'd own this is she's not

**Comment by oliviabrown9** [comment]
@flodolo all the above questions for @st3fan still stand. I think he’s the point person for localization on this. That being said, I don’t think these strings should block release since they are only for Siri.

**Comment by Sdaswani** [comment]
@sblatz can you address the change requests please? Or is this something Stefan needs to do?

**Comment by oliviabrown9** [comment]
@sdaswani the issue is the script that we have, which I think Stefan owns.

**Comment by Sdaswani** [comment]
Ah @oliviabrown9 that lovely script - @sblatz if you can take a go at it doesn't hurt to spend an hour, but if you think if @st3fan is better suited that's fine too.

**Comment by flodolo** [comment]
I don't believe it's the script. We seem to be hitting an issue where `xcodebuild` (XCode command line tool) doesn't export the string ID, or the string ID is not set correctly in the code.

["Erase browsing session" string](https://github.com/mozilla-mobile/focus-ios/blob/v7.0/Blockzilla/Intents.intentdefinition#L21). Looks like that's a new file added in XCode 10?

Maybe there's no way to have a [proper string ID for those](https://forums.developer.apple.com/thread/107412), but I'm not an Xcode developer.

**Comment by sblatz** [comment]
> I also noticed that this file has a different file path (Blockzilla/en.lproj/... vs Blockzilla/...) than other strings files. I'm not sure if this is contributing to any of the issues.

This was an issue we had in 6.0 which led to [this PR](https://github.com/mozilla-mobile/ios-l10n-scripts/commit/775a50d5a52514e6340646df5fcb38e2b4e20489) by @st3fan to fix the script. 

I have submitted a new PR to fix it [found here](https://github.com/mozilla-mobile/ios-l10n-scripts/pull/42). I believe this should resolve the export issue :)

**Comment by st3fan** [comment]
I filed this issue to bring the export scripts up to date with Xcode 10

https://github.com/mozilla-mobile/ios-l10n-scripts/issues/43

That will fix the missing comments issue.

**Comment by flodolo** [comment]
I assumed we can close this.

**Pull request closed without merging**